packages/config/src/android/Updates.ts

```diff
@@ -0,0 +1,103 @@
+import { ExpoConfig } from '../Config.types';
+import { Document } from './Manifest';
+
+export function getUpdateUrl(config: ExpoConfig, username: string | null) {
+  const user = typeof config.owner === 'string' ? config.owner : username;
+  if (!user) {
+    return null;
+  }
+  return `https://exp.host/@${user}/${config.slug}`;
+}
+
+export function getSDKVersion(config: ExpoConfig) {
+  return typeof config.sdkVersion === 'string' ? config.sdkVersion : null;
+}
+
+export function getUpdatesEnabled(config: ExpoConfig) {
+  return config.updates?.enabled !== false;
+}
+
+export function getUpdatesTimeout(config: ExpoConfig) {
+  return config.updates?.fallbackToCacheTimeout ?? 0;
+}
+
+export function getUpdatesCheckOnLaunch(config: ExpoConfig) {
```

Comment:
isn't `WIFI_ONLY` one of the options as well?

Reply:
it is in the new API but it has no analog in the managed config currently, so there's nothing that maps to it yet